libpod/runtime.go

```diff
@@ -69,6 +70,7 @@ type Runtime struct {
 	conmonPath     string
 	valid          bool
 	lock           sync.RWMutex
+	ImageRuntime   *image.Runtime
```

Comment:
Instead of making this public, can there be an accessor? func ImageRuntime() (*image.Runtime)

Reply:
yup bet, fixed